tests/test_sound.py

```diff
@@ -1,6 +1,6 @@
 from music21 import *
 
 def test_midi():
-    result = converter.parse('tests/sample.musicxml').write('midi')
+    result = converter.parse('tests/testFiles/Chant.musicxml').write('midi')
```

Comment:
ditto.

Reply:
same idea, there won't be any `file conflicts'.